src/core/base/root.ts

```diff
@@ -1,6 +1,7 @@
-import {bind, EventUtils, ObserverCallback, ESLMediaRuleList} from '@exadel/esl';
+import {bind, EventUtils, ESLMediaRuleList} from '@exadel/esl';
```

Comment:
Cosmetic: that might be better to use exact paths (or at least make all exact(`@exadel/esl/modules/..`) or global (`@exadel/esl`))

Reply:
so, should it be like now?

Reply:
Yep, now everything looks fine
Thanks!